ui/client/containers/BaseProcesses.js

```diff
@@ -89,22 +89,26 @@ class BaseProcesses extends PeriodicallyReloadingComponent {
     const query = _.pick(queryString.parse(window.location.search), this.searchItems || [])
     const data = Object.assign(query, search, this.queries || {})
 
-    HttpService.fetchProcesses(data).then (fetchedProcesses => {
+    HttpService.fetchProcesses(data).then (response => {
       if (!this.state.showAddProcess) {
-        this.setState({processes: fetchedProcesses, showLoader: false})
+        this.setState({processes: response.data, showLoader: false})
       }
     }).catch(() => this.setState({ showLoader: false }))
   }
 
   reloadStatuses() {
-    HttpService.fetchProcessesStatus().then (statuses => {
+    HttpService.fetchProcessesStatus().then (response => {
       if (!this.state.showAddProcess) {
-        this.setState({ statuses: statuses, showLoader: false, statusesLoaded: true })
+        this.setState({ statuses: response.data, showLoader: false, statusesLoaded: true })
       }
     }).catch(() => this.setState({ showLoader: false }))
   }
 
   retrieveSelectedCategories(categories) {
+    if (!_.isArray(categories)) {
```

Comment:
is it really needed? can we fix it in invocation? Here it seems like another js hack :/

Reply:
Ekhm.. You can put at query params somethink like this: categories=cat1,cat2 and then this will be array. But when you put categories=cat1 then it is string.. :) Yep, this is js :)